programs/witness_node/main.cpp

```diff
@@ -239,11 +258,21 @@ int main(int argc, char** argv) {
             data_dir = fc::current_path() / data_dir;
       }
 
+      // load witness node initial configuration
       fc::path config_ini_path = data_dir / "config.ini";
       if( !fc::exists(config_ini_path) )
          create_new_config_file( config_ini_path, data_dir, cfg_options );
       load_config_file( config_ini_path, cfg_options, options );
 
+      // load witness node logging configuration
+      const auto logging_ini_path = data_dir / "logging.ini";
+      if (!exists(logging_ini_path))
+      {
+         create_logging_config_file (logging_ini_path, data_dir);
```

Comment:
What about the logging parameters that already exist in the config.ini?

Reply:
that's right, our change doesn't handle it because we do not need to consider such use case, I can help with implementation if confirmed this is the way to go - I see some discussions still pending in #149

Reply:
I believe the consensus of #149 is:

```
If (config.ini does not exist)
{
    create config.ini;
    create logging.ini;
}

Parse config.ini for application configuration;

If (logging.ini exists)
{
    Parse logging.ini for logging configuration;
} 
else 
{
    Parse config.ini for logging configuration;
}
```
@pmconrad @abitmore @ryanRfox please verify that consensus has been reached, and my pseudo-code meets the requirements.

Reply:
Yes, @jmjatlanta I agree with your pseudo-code above. Build that for #149.

Notes for a future Enhancement Request:
* Consider versioning the .ini files to match the Release ID
* When the software starts and encounters a legacy .ini file, it may be beneficial to warn/log/notify the user/system of <these new configuration parameters> and their impacts using a legacy .ini file with this Release.

Reply:
the implementation extended to include the case when logging parameters already exist in the config.ini